akka-actor-typed/src/main/scala/akka/actor/typed/internal/receptionist/LocalReceptionist.scala

```diff
@@ -31,6 +32,8 @@ private[akka] final class LocalReceptionist
 @InternalApi
 private[akka] object LocalReceptionist extends ReceptionistBehaviorProvider {
 
+  override val name = "local-receptionist"
```

Comment:
I think we mostly use the camel case naming convention for actor names (`localReceptionist`)

Reply:
🐫